src/content/recipes/@emotion/styled.md

```diff
@@ -29,44 +29,14 @@ If you’d like to see the example code of this recipe, check out the [example r
 
 ![Completed Emotion example with theme switcher](https://user-images.githubusercontent.com/18172605/208312563-875ca3b0-e7bc-4401-a445-4553b48068ed.gif)
 
-## Install `@storybook/addon-styling`
+## Install `@storybook/addon-themes`
 
-Add the `@storybook/addon-styling` package to your DevDependencies
+To get started, you'll need to install [`@storybook/addon-themes`](https://storybook.js.org/addons/@storybook/addon-themes).
 
-```shell
-yarn add -D @storybook/addon-styling
-```
-
-## Auto-config
-
-<div class="aside">
-
-<span aria-hidden="true">📣</span> Before running this codemod, please ensure that you have no other changes in your git branch.
-
-</div>
-
-As of version 1.3, `@storybook/addon-styling` offers a codemod for to automatically configure your storybook with Emotion.
-
-To try it out, run the following script:
+Run the following script to install and register the addon:
 
-```shell
-# Run the postinstall script from the root of your project
-yarn addon-styling-setup
-```
-
-If the codemod didn't work, please let us know in [this GitHub issue](https://github.com/storybookjs/addon-styling/issues/49#issue-1746365130) so we can continue to make the codemod even better. In the meantime, the manual instructions below will get you up and running in no time.
-
-## Manual
-
-### Register the addon
-
-Register the addon with Storybook in `.storybook/main.js`.
-
-```js
-module.exports = {
-  stories: ['../stories/**/*.mdx', '../stories/**/*.stories.@(js|jsx|ts|tsx)'],
-  addons: ['@storybook/addon-essentials', '@storybook/addon-styling'],
-};
+```bash
```

Comment:
@Integrayshaun one question for you about this pull request. I really liked the fact that you went with the `add` option to install it, but I was wondering, we don’t actually have the option documented, nor the workflow for it (snippets, etc) ,  can users still manually register it or run the codemod to get it set up? If so, we could at least have a callout mentioning it for all the set up instructions. WDYT?

Reply:
@jonniebigodes I think that's a good call out. What sorts of changes do you think we need to make across the docs to make `storybook add` more widely known?

Users can still manually install and register the addons if they'd like. I just tried to limit the amount of info dumped into the recipes

Reply:
@Integrayshaun I'm working through them and will follow up with you on them.  Sounds good?